packages/client-config/src/client-config-schema/client_config_v1.2.ts

```diff
@@ -0,0 +1,272 @@
+      scopes: string[];
+      /**
+       * URIs used to redirect after signing in using an identity provider
+       *
+       * @minItems 1
+       */
+      redirect_sign_in_uri: string[];
+      /**
+       * URIs used to redirect after signing out
+       *
+       * @minItems 1
+       */
+      redirect_sign_out_uri: string[];
+      response_type: 'code' | 'token';
+    };
+    /**
+     * Cognito User Pool standard attributes required for signup
+     *
+     * @minItems 0
+     */
+    standard_required_attributes?: AmazonCognitoStandardAttributes[];
+    /**
+     * Cognito User Pool username attributes
+     *
+     * @minItems 1
+     */
+    username_attributes?: ('email' | 'phone_number' | 'username')[];
+    user_verification_types?: ('email' | 'phone_number')[];
+    unauthenticated_identities_enabled?: boolean;
+    mfa_configuration?: 'NONE' | 'OPTIONAL' | 'REQUIRED';
+    mfa_methods?: ('SMS' | 'TOTP')[];
+  };
+  /**
+   * Outputs generated from defineData
+   */
+  data?: {
+    aws_region: AwsRegion;
+    /**
+     * AppSync endpoint URL
+     */
+    url: string;
+    /**
+     * generated model introspection schema for use with generateClient
+     */
+    model_introspection?: {
+      [k: string]: unknown;
+    };
+    api_key?: string;
+    default_authorization_type: AwsAppsyncAuthorizationType;
+    authorization_types: AwsAppsyncAuthorizationType[];
+  };
+  /**
+   * Outputs manually specified by developers for use with frontend library
+   */
+  geo?: {
+    /**
+     * AWS Region of Amazon Location Service resources
+     */
+    aws_region: string;
+    /**
+     * Maps from Amazon Location Service
+     */
+    maps?: {
+      items: {
+        [k: string]: AmazonLocationServiceConfig;
+      };
+      default: string;
+    };
+    /**
+     * Location search (search by places, addresses, coordinates)
+     */
+    search_indices?: {
+      /**
+       * @minItems 1
+       */
+      items: string[];
+      default: string;
+    };
+    /**
+     * Geofencing (visualize virtual perimeters)
+     */
+    geofence_collections?: {
+      /**
+       * @minItems 1
+       */
+      items: string[];
+      default: string;
+    };
+  };
+  /**
+   * Outputs manually specified by developers for use with frontend library
+   */
+  notifications?: {
+    aws_region: AwsRegion;
+    amazon_pinpoint_app_id: string;
+    /**
+     * @minItems 1
+     */
+    channels: AmazonPinpointChannels[];
+  };
+  /**
+   * Outputs generated from defineStorage
+   */
+  storage?: {
+    aws_region: AwsRegion;
+    bucket_name: string;
+    buckets?: AmplifyStorageBucket[];
+  };
+  /**
+   * Outputs generated from backend.addOutput({ custom: <config> })
+   */
+  custom?: {
+    [k: string]: unknown;
+  };
+}
+/**
+ * This interface was referenced by `undefined`'s JSON-Schema definition
+ * via the `patternProperty` ".*".
+ */
+export interface AmazonLocationServiceConfig {
+  /**
+   * Map style
+   */
+  style?: string;
+}
```

Comment:
It seems the name property is missing from this type?

Reply:
I might have misconfigured or am using a different version of `json-schema-to-typescript`. The closest I would get to the existing client config was running:
`json2ts packages/client-config/src/client-config-schema/schema_v1.2.json -o packages/client-config/src/client-config-schema/client_config_v1.2.ts --style.singleQuote --ignoreMinAndMaxItems
`

I'll update this manually.